dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -405,8 +407,27 @@ object LagomPlugin extends AutoPlugin {
   }
 
   private def runAllMicroservicesTask: Initialize[Task[Unit]] = Def.taskDyn {
+
     val projects = microservicesProjects.value
-    val filter = ScopeFilter(inProjects(projects: _*))
+
+    // check if application.conf is present in service
+    def isValidProject(p: ProjectRef) = {
+
+      val prj = Project.getProject(p, buildStructure.value)
+
+      val r: PathFinder = {
+        (new File(prj.get.base.getPath) / "src") ** "application.conf"
```

Comment:
`resourceDirectories` is a sbt setting defined for (and hence available to) any sbt project. Enter the sbt console of *any* project, and type `show resourceDirectories`.